agent/pom.xml

```diff
@@ -119,6 +124,18 @@
             <goals>
               <goal>shade</goal>
             </goals>
+            <configuration>
```

Comment:
what is this for?

Reply:
There are apparently signed jar dependencies that cause the shading to fail with a security error. The filter removes the signed artifacts from the jars its bundling so that the JVM will actually run it.